recipes/cargo-llvm-cov/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set version = "0.6.9" %}
+
+package:
+  name: cargo-llvm-cov
+  version: {{ version }}
+
+source:
+  url: https://github.com/taiki-e/cargo-llvm-cov/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 0a1b63922cfb6f2526a61bc5e13868c3f3345aa3af7337df07a69eab3d4b243e
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}        # [unix]
+    - {{ compiler('m2w64_c') }}  # [win]
```

Any particular reason why you want the m2 compilers on windows? Why not simply:
```suggestion
    - {{ compiler('c') }}
```

---

Not really, no. I copied this revipe from taplo and, once it worked, didn't bother to find the minimum set of build dependencies 😬 I can try removing it 👍🏼